src/Cake.Tests/Cake.Tests.csproj

```diff
@@ -23,6 +23,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <LangVersion>default</LangVersion>
```

Comment:
As there's a PR for  ` <LangVersion>5</LangVersion>` this change feels unnecessary, right?